### DISCUSSION TITLE
Is there a way to get the estimated generation time through the API?

### DISCUSSION THREAD

**Discussion by monis24** [comfyanonymous/ComfyUI#1280] [Q&A]
I'm writing a python script to make API calls to generate images and I'm wondering if there's any way for me to get the estimated time it'll take to generate my images?

**Comment by ntdviet** [comment]
No, but you can traverse through the nodes and estimate the time based on samplers and vae decoders. Guess if you can do it, it would be a not trivial achievement :)
